tests/virt/cluster/common_templates/centos/test_centos_os_support.py

```diff
@@ -30,6 +30,7 @@
 
 
 class TestCommonTemplatesCentos:
+    @pytest.mark.gating
```

Comment:
we have rhel custom templates in gating, why we need centos?

Reply:
+1
no need to add another common templates tests